apolloschurchapp/ios/apolloschurchapp/apolloschurchapp.entitlements

```diff
@@ -6,7 +6,7 @@
 	<string>development</string>
 	<key>com.apple.developer.associated-domains</key>
 	<array>
-		<string>applink:$(APP_DATA_URL)</string>
+		<string>applinks:apollos-church-api.herokuapp.com</string>
```

Comment:
I don't like this because if we forget to change it (or someone in the open-source community forgets) it's wrong and would cause problems. I'd rather leave it off and add it to the docs unless we can figure out another way to make it universal